pkg/api/store/workload/toleration_test.go

```diff
@@ -0,0 +1,40 @@
+package workload
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_getSchedulingRule(t *testing.T) {
```

Comment:
I think the function name should be pascal cased

Reply:
As my test target is `schedulingRuleFoundLinux` function, I rename it as `Test_schedulingRuleFoundLinux`.

Reply:
I meant it should be of the form `TestSchedulingRuleFoundLinux`